three.c

```diff
@@ -0,0 +1,77 @@
+//Ashank Kumar, program takes in three intigers, checks if they are valid, checks if they form a triangle, and what type of triangle
+#include <stdio.h>
+
+int isTriangle(int a, int b, int c)
+{
+	if((a+b > c) && (a+c > b) && (c+b > a)) //doesnt work
```

Comment:
yeah, so you already know that this doesn't work properly - but in addition to that, you should include comments for each function.